draft_migration.sql

```diff
@@ -0,0 +1,61 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+SET search_path = events_api, public;
+
+CREATE TABLE IF NOT EXISTS webhooks (
+                                        id BIGSERIAL NOT NULL,
```

Comment:
most of our other API's have standardized on using KSUID's for primary keys, you can define this as a char(27) in the db.